### ISSUE TITLE
kendo-sortable prevents `mouseenter` events

### ISSUE THREAD

**Issue by maximedupre** [telerik/kendo-angular#2027]
## I'm submitting a...
* Regression report (Works in `v1.1.1`. Stopped working in `2.0.0`)

## Current behavior

When a `kendo-sortable` component is added, the `mouseenter` event isn't fired anymore when the "mouse is down" (`mousedown`) in other unrelated components. 

For example I could have another component in my app that is a sibling to a `kendo-sortable`. In this component there would be elements bound to the `mousedown` and `mouseenter` events. When the mouse is down and the cursor enters an element, `mouseenter` is not fired. If the mouse isn't down, `mouseenter` is fired just fine.

I have isolated the problem to the minimum and it is because the `kendo-sortable` component adds a `pointerdown` event on `document`. Unbinding/removing this event listener manually with the dev tool solves the problem. Removing the `kendo-sortable` from my application also removes the undesired event listener.

## Expected behavior
When a `kendo-sortable` component is in my application, `mouseenter` should still be fired when the mouse is down.

## Environment
Package versions:
```
├── @angular-devkit/build-angular@0.12.2
├── @angular/animations@7.2.1
├── @angular/cli@7.2.2
├── @angular/common@7.2.1
├── @angular/compiler@7.2.1
├── @angular/compiler-cli@7.2.1
├── @angular/core@7.2.1
├── @angular/forms@7.2.1
├── @angular/http@7.2.1
├── @angular/language-service@7.2.1
├── @angular/platform-browser@7.2.1
├── @angular/platform-browser-dynamic@7.2.1
├── @angular/router@7.2.1
├── @fortawesome/angular-fontawesome@0.3.0
├── @fortawesome/fontawesome-svg-core@1.2.12
├── @fortawesome/free-brands-svg-icons@5.6.3
├── @fortawesome/free-regular-svg-icons@5.6.3
├── @fortawesome/free-solid-svg-icons@5.6.3
├── @progress/kendo-angular-buttons@4.3.2
├── @progress/kendo-angular-dialog@3.10.1
├── @progress/kendo-angular-dropdowns@3.4.2
├── @progress/kendo-angular-inputs@3.5.0
├── @progress/kendo-angular-intl@1.6.1
├── @progress/kendo-angular-l10n@1.3.0
├── @progress/kendo-angular-layout@3.2.0
├── @progress/kendo-angular-popup@2.5.0
├── @progress/kendo-angular-sortable@2.0.1
├── @progress/kendo-angular-upload@4.2.2
├── @progress/kendo-theme-default@2.58.0
├── @telerik/kendo-intl@1.4.4
├── @types/bootstrap@4.2.0
├── @types/dragula@2.1.34
├── @types/gapi@0.0.36
├── @types/gapi.auth2@0.0.48
├── @types/jasmine@3.3.5
├── @types/jasminewd2@2.0.6
├── @types/jquery@3.3.29
├── @types/jquery-knob@1.2.31
├── @types/node@10.12.18
├── @types/pdfjs-dist@0.1.2
├── audiobuffer-to-wav@1.0.0
├── bootstrap@4.2.1
├── core-js@2.6.2
├── dragula@3.7.2
├── ffmpeg.js@3.1.9001
├── interact.js@1.2.8
├── is-wav@1.1.2
├── jasmine-core@3.3.0
├── jasmine-marbles@0.4.1
├── jasmine-spec-reporter@4.2.1
├── jquery@3.3.1
├── jszip@3.1.5
├── jwt-decode@2.2.0
├── karma@3.1.4
├── karma-chrome-launcher@2.2.0
├── karma-coverage-istanbul-reporter@2.0.4
├── karma-jasmine@2.0.1
├── karma-jasmine-html-reporter@1.4.0
├── karma-scss-preprocessor@4.0.0
├── lamejs@1.2.0
├── midiconvert@0.4.5 (github:maximedupre/MidiConvert#708356e4159ee9e8e5980c74b9a7a7f2b42da331)
├── moment@2.23.0
├── ng2-pdf-viewer@5.2.3
├── ng2-trim-directive@2.3.2
├── ngx-moment@3.3.0
├── open-iconic@1.1.1
├── pdfjs-dist@2.0.943
├── perfect-scrollbar@1.4.0
├── protractor@5.4.2
├── rxjs@6.3.3
├── rxjs-compat@6.3.3
├── soundcloud@3.3.1
├── tone@13.4.9
├── ts-helpers@1.1.2
├── ts-node@7.0.1
├── tslib@1.9.3
├── typescript@3.2.2
├── videoconverter@0.1.0
├── webmidi@2.3.1
├── ws@6.1.2
└── zone.js@0.8.28
```

Browser:
- Chrome (desktop) version 71

System:
- TypeScript version: 3.2.2
- Node version: 11.6.0
- Platform:  Mac Mojave

**Comment by ipeshev** [comment]
@maximedupre thank you for the bug report we will investigate it further, reproduction demo [here](https://stackblitz.com/edit/angular-t8ucmo).

**Comment by danielkaradachki** [comment]
Fixed in 2.1.0:

https://stackblitz.com/edit/angular-t8ucmo-by15px?file=app/app.component.ts
